\hypertarget{classpyrst_1_1pyrst_1_1exceptions_1_1_connection_exception}{}\section{pyrst.\+pyrst.\+exceptions.\+Connection\+Exception Class Reference}
\label{classpyrst_1_1pyrst_1_1exceptions_1_1_connection_exception}\index{pyrst.\+pyrst.\+exceptions.\+Connection\+Exception@{pyrst.\+pyrst.\+exceptions.\+Connection\+Exception}}
Inheritance diagram for pyrst.\+pyrst.\+exceptions.\+Connection\+Exception\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=2.000000cm]{classpyrst_1_1pyrst_1_1exceptions_1_1_connection_exception}
\end{center}
\end{figure}
\subsection*{Public Member Functions}
\begin{DoxyCompactItemize}
\item 
\hypertarget{classpyrst_1_1pyrst_1_1exceptions_1_1_connection_exception_a656fe7d3628524a52c21291116affe1a}{}def {\bfseries \+\_\+\+\_\+repr\+\_\+\+\_\+} (self)\label{classpyrst_1_1pyrst_1_1exceptions_1_1_connection_exception_a656fe7d3628524a52c21291116affe1a}

\end{DoxyCompactItemize}


The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
pyrst/exceptions.\+py\end{DoxyCompactItemize}
